test/pt2_to_circle_test/builder.py

```diff
@@ -189,6 +190,25 @@ def _run(
             torch_result = [res for res in torch_result if res is not None]
             validate_result(torch_result, circle_result, **self.tolerance)
 
+        if dynamic_shapes:
```

Comment:
I think this should be placed after `verify_circle` or inside that function for early-stopping.